### ISSUE TITLE
customvisualization is not added in schema of component.

### ISSUE THREAD

**Issue by harsh1868** [TIBCOSoftware/jasperreports#209]
Here I found that customvisualization is not added in the schema of componentElement, and hence whenever we try to add the same in jrxml using maven it's giving an error that this element is not allowed here.

**Comment by harsh1868** [comment]
I'm facing this problem only when I try report generation using maven dependency but it's working fine with jasper soft studio.

**Comment by teodord** [comment]
Do you have the jasperreports-custom-visualization-xxx.jar in your classpath?
https://repo1.maven.org/maven2/net/sf/jasperreports/jasperreports-custom-visualization/6.17.0/

**Comment by harsh1868** [comment]
@teodord Thank you for your response, but I want to tell you that I already add this jar file using maven dependency only, but still it was not working. What I found is that actually in the schema of the component entity we don't have any declaration for custom component and hence when we try to the same jrxml file which jasper soft has generated in our java project it's giving error. So might it's problem from top level. That is what I observed
